src/frontend/components/dashboard/MapTab/MapControls.jsx

```diff
@@ -72,7 +72,7 @@ const MapControls = ({
           onChange={onGeographyChange}
           options={[
             { label: 'County boundaries', value: 'counties' },
-            { label: 'Census blocks', value: 'blocks' },
+            { label: 'Census tracts', value: 'tracts' },
```

Comment:
There are like ~70K tracts and ~12M blocks in the country so let's start with tracts and determine if blocks are useful.